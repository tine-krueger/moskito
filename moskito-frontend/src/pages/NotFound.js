import Error404 from "../components/404/Error404";
import Logo from "../components/Logo/Logo";

export default function NotFound() {
    return(
        <>
            <Logo/>
            <Error404/>
        </>
    )
}