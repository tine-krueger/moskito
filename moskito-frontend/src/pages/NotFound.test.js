import { render } from '@testing-library/react'
import 'jest-styled-components'
import NotFound from './NotFound'

describe('Not Found', () => {
  it('renders correctly', () => {
    const { container } = render(<NotFound/>)
    expect(container).toMatchSnapshot()
  })
})