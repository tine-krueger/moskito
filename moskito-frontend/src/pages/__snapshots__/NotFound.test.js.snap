// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found renders correctly 1`] = `
.c3 {
  padding: 1em 2em;
  font-family: 'Arvo';
  font-size: 1.1rem;
  border-radius: 10px;
  width: auto;
  border: none;
  background: var(--background);
  box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6),-2px -2px 2px 0px rgba(255,255,255,0.5);
  color: var(--headline);
  text-transform: uppercase;
  margin: 1em auto 3em auto;
}

.c3:hover {
  box-shadow: inset 4px 4px 6px 0 rgba(0,0,0,0.2),inset -3px -4px 6px 0 rgba(255,255,255,0.3);
  outline: none;
}

.c2 {
  margin: 6em 2.5em;
}

.c0 {
  padding-top: 2em;
  text-align: center;
}

.c1 {
  box-shadow: inset 4px 4px 6px 0 rgba(0,0,0,0.2),inset -3px -4px 6px 0 rgba(255,255,255,0.3);
  border-radius: 20px;
  padding: 1em 2em;
  width: 85%;
}

<div>
  <div
    class="c0"
    data-testid="logo"
  >
    <img
      alt="Moskito Logo"
      class="c1"
      src="moskitoLogo.svg"
    />
  </div>
  <div
    class="c2"
  >
    <h2>
      Fehler 404: Ups, hier stimmt was nicht!
    </h2>
    <p>
      Diese Seite existiert leider nicht (mehr)!
    </p>
    <button
      class="c3"
    >
      Zur√ºck
    </button>
  </div>
</div>
`;
